Tests/WebAuthnTests/WebAuthnManagerTests+Authentication.swift

```diff
@@ -0,0 +1,188 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the WebAuthn Swift open source project
+//
+// Copyright (c) 2022 the WebAuthn Swift project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of WebAuthn Swift project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+@testable import WebAuthn
+import XCTest
+import SwiftCBOR
+import Crypto
+
+// swiftlint:disable line_length
+
+extension WebAuthnManagerTests {
```

Comment:
We need to double check that these are picked up by test discovery on Linux because IIRC tests in extensions didn't used to be

Reply:
It did discover it on Linux, but I moved it to a separate class to keep the tests clean.